docs/resources/project_role.md

```diff
@@ -0,0 +1,41 @@
+---
+page_title: "doppler_project_role Resource - terraform-provider-doppler"
+subcategory: ""
+description: |-
+	Manage a Doppler project role.
+---
+
+# doppler_project_role (Resource)
+
+Manage a Doppler project_role.
+
+## Example Usage
+
+```terraform
+resource "doppler_project_role" "log_viewer" {
+  name        = "Log Viewer"
+  permissions = ["enclave_config_logs"]
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) The name of the Doppler project role
+- `permissions` (List of String) A list of Doppler project permissions
```

Comment:
Can we link to the [API docs](https://docs.doppler.com/reference/project_roles-create) that contain the list of valid permission values here? Would be a nice QoL thing for anyone looking at the TF docs.

Reply:
Good call 👍